config/dlss_baseline.yml

```diff
@@ -84,15 +84,27 @@ Style/BlockComments:
 Style/ClassAndModuleChildren:
   Enabled: false
 
+Style/ClosingParenthesisIndentation:
+  Enabled: false
```

Comment:
Good points.  Does your last example pass rubocop?  Or does it complain b/c parens are not aligned, meaning we need:

```ruby
expect(result.to_xml).to be_equivalent_to
  (
    <<-EOF
      <fileGroupDifference>
        <subset change="identical" count="3">
       ...
    EOF
  )
```